### ISSUE TITLE
TLDR update error

### ISSUE THREAD

**Issue by shev85** [tldr-pages/tldr#14313]
tldr --update
⠧ Updating.../usr/local/lib/node_modules/tldr/node_modules/adm-zip/adm-zip.js:62
            throw Utils.Errors.INVALID_FILENAME();
            ^

Error: ADM-ZIP: Invalid filename
    at Object.INVALID_FILENAME (/usr/local/lib/node_modules/tldr/node_modules/adm-zip/util/errors.js:56:16)
    at new module.exports (/usr/local/lib/node_modules/tldr/node_modules/adm-zip/adm-zip.js:62:32)
    at WriteStream.<anonymous> (/usr/local/lib/node_modules/tldr/lib/remote.js:32:21)
    at WriteStream.emit (node:events:529:35)
    at finish (node:internal/streams/writable:748:10)
    at finishMaybe (node:internal/streams/writable:733:9)
    at afterWrite (node:internal/streams/writable:507:3)
    at onwrite (node:internal/streams/writable:480:7)
    at node:internal/fs/streams:465:5
    at node:internal/fs/streams:421:7

Node.js v18.19.1

dosen’t work，how to fix it?
